### PR TITLE
Electric Guitar soundrewrite and mapping

### PR DESCRIPTION
**Pull request by Niknakflak**
It's still loud and ear hurting but now it's in tune and shorter. You're welcome.  Made for Okand's birthday.  Now it maps the abomination as well so people can finally shred without admin help to spawn.  The range is now shorter with less notes but it plays all the songs with no issues.  Sounds were generated with a digital audio workstation and modified by me a lot.

For comparison:
New sounds playing the same thing as the old sounds were playing: 
https://file.house/q2fv.mp4
Note, this was before the sounds were normalized to reduce feedback.

Old sounds for reference: 
https://file.house/dVtz.mp4

### CODE REVIEW

File: _maps/map_files/EfficiencyStation/EfficiencyStation.dmm [hunk 1]
```diff
@@ -56874,9 +56874,13 @@
 	},
 /area/shuttle/syndicate)
 "ckj" = (
-/obj/machinery/portable_atmospherics/canister/bz,
```
Comment (Niknakflak) [inline _maps/map_files/EfficiencyStation/EfficiencyStation.dmm]: I'm not sure what happened here

**Comment by TechnoAlchemisto** [comment]
loud and hurts my ears

File: _maps/map_files/EfficiencyStation/EfficiencyStation.dmm (hunk 1 above)
Comment (xxalpha) [inline _maps/map_files/EfficiencyStation/EfficiencyStation.dmm]: Eco friendly map merger

**Comment by WJohn** [comment]
Wow that's a huge improvement 👍

**Comment by HankMoodyExperience** [comment]
good shit, now make a space accordion for my realistic remove kebab roleplay

File: _maps/map_files/EfficiencyStation/EfficiencyStation.dmm (hunk 1 above)
Comment (ma44) [inline _maps/map_files/EfficiencyStation/EfficiencyStation.dmm]: Seems undocumented

**Comment by Niknakflak** [comment]
Making instrument sound files takes awhile because you have to splice them all individually and its a pain.

**Comment by Niknakflak** [comment]
I broke something because the buildtools failed 3 times, dont know what though.  Let me know.

**Comment by AnturK** [comment]
Was just some Travis hiccup, checks out after cache purge and restart.

**Comment by Cheridan** [comment]
I think I prefer the old one, the notes had better sustain so it had a more organic/realistic electric guitar sound.

**Comment by Niknakflak** [comment]
The old one wasnt even in tune.  Every single old sound was generated from a single recorded note and pitch changed.  The new sounds are all individually recorded and everyone I showed it to says the new ones are wayyyy better.

**Pull request merged**